src/Components/BehaviorsModal/Internal/Widgets/DataHistoryWidget/Internal/DataHistoryWidgetErrorHandling.tsx

```diff
@@ -0,0 +1,188 @@
+import {
+    Callout,
+    IStyleFunctionOrObject,
+    Link,
+    Text,
+    useTheme
+} from '@fluentui/react';
+import { useBoolean, useId } from '@fluentui/react-hooks';
+import React, { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import ConnectionErrorImg from '../../../../../../Resources/Static/connectionError.svg';
+import PermissionErrorImg from '../../../../../../Resources/Static/priviledgedAccess.svg';
+import GenericErrorImg from '../../../../../../Resources/Static/noResults.svg';
+import { DOCUMENTATION_LINKS } from '../../../../../../Models/Constants/Constants';
+import { IComponentError } from '../../../../../../Models/Constants/Interfaces';
+import { getCardboardListCalloutStyles } from '../../../../../CardboardListCallout/CardboardListCallout.styles';
```

Comment:
why do we bring these styles from some other component? If they need to be generic shouldn't we pull them out to a shared file instead of referencing another component directly? If we make chagnes there I would never think to come look here to see if I broke something.

Reply:
copied the used styling to the shared file and importing from there instead.